core/src/main/java/org/geysermc/geyser/registry/populator/RecipeRegistryPopulator.java

```diff
@@ -173,7 +173,7 @@ private static RecipeData getCraftingDataFromJsonNode(JsonNode node, Int2ObjectM
             /* Convert end */
 
             return ShapedRecipeData.shaped(uuid.toString(), shape.get(0).length(), shape.size(),
-                    inputs.stream().map(ItemDescriptorWithCount::fromItem).toList(), Collections.singletonList(output), uuid, "crafting_table", 0, netId);
+                    inputs.stream().map(ItemDescriptorWithCount::fromItem).toList(), Collections.singletonList(output), uuid, "crafting_table", 0, netId, true);
```

Comment:
I wonder if we should make this false -
might resolve this issue:
https://github.com/GeyserMC/Geyser/issues/4103

Reply:
Maybe